stdlib/public/core/CompilerProtocols.swift

```diff
@@ -194,10 +194,46 @@ public protocol _ExpressibleByBuiltinIntegerLiteral {
   init(_builtinIntegerLiteral value: _MaxBuiltinIntegerType)
 }
 
-/// Conforming types can be initialized with integer literals.
+/// A type that can be initialized with an integer literal.
```

Comment:
It is somewhat disconnected from the integer protocols. Is it possible to move this into master to avoid merge conflicts in the future?